Q/Qt/build_tarballs.jl

```diff
@@ -217,17 +217,17 @@ products_macos = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     BuildDependency("Xorg_libX11_jll"),
```

Comment:
This is still build-only?

Reply:
Yes, that one is certainly always installed on an X system. I'm nervous about including it, since I'm not sure what happens if the BB libX11 is loaded by something that renders on a screen driven by the system libX11?

Reply:
GTK.jl has it and I don't remember someone reporting problems like this one. Note als that libx11 has a couple of runtime depencies: https://github.com/JuliaPackaging/Yggdrasil/blob/master/X/Xorg_libX11/build_tarballs.jl#L40-L41, so by the same argument also libxcb should be a build-only dependency.

If you want we can try to keep it this way, and we'll see what happens in #1282

Reply:
OK, I prefer to keep it like this for now, especially since adding deps is easier than removing them for the registry.